mods/ra/maps/allies-01/map.yaml

```diff
@@ -539,9 +539,12 @@ Rules:
 			GenericVisibility: Enemy, Ally, Neutral
 			GenericStancePrefix: false
 			ShowOwnerRow: false
-	^CivInfantry:
+	^ArmedCivilian:
 		RevealsShroud:
 			Range: 0c0
```

Comment:
This doesn't work anymore.
![civreveal](https://cloud.githubusercontent.com/assets/7704140/8508396/940284ea-226e-11e5-90f8-e266221ab129.png)